### PR TITLE
[PLAYER-5181] Chromecast list screen created

### PR DESCRIPTION
**Pull request by isheftel-ooyala**
Chromecast list screen created

### CODE REVIEW

**Review by SergeyBicarte — APPROVED** [review]
LGTM

File: sdk/react/ooyalaSkinPanelRenderer.js [hunk 1]
```diff
@@ -95,6 +96,20 @@ OoyalaSkinPanelRenderer.prototype.renderErrorScreen = function() {
...
 };
 
+OoyalaSkinPanelRenderer.prototype.renderCastScreen = function() {
+  return (
+    <CastDevicesScreen
+      height={this.skin.state.height}
+      width ={this.skin.state.width}
+      onDismiss={() => this.core.dismissOverlay()}
+      config={{
+        moreDetailsScreen: this.skin.props.moreOptionsScreen,
```
Comment (atopilskii) [inline sdk/react/ooyalaSkinPanelRenderer.js]: Why do we use this "moreOptionsScreen" for cast button?

File: sdk/react/panels/CastDevicesScreen.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+      }
+    ).start(this.onDismissBtnPress);
+  };
+
+  render() {
+    const dismissButton = Utils.renderRectButton(BUTTON_NAMES.DISMISS,
+      styles.iconDismiss,
+      this.props.config.icons.dismiss.fontString,
+      this.onDismissPress, dismissButtonSize,
+      this.props.config.moreDetailsScreen.color,
```
Comment (atopilskii) [inline sdk/react/panels/CastDevicesScreen.js]: The same thing: i think it is necessery to use new color for Cast. What do you think?

File: sdk/react/panels/CastDevicesScreen.js [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+      this.props.config.icons.dismiss.fontFamilyName);
+
+    const castButton = Utils.renderRectButton(BUTTON_NAMES.CAST,
+      null,
+      this.props.config.icons.cast.fontString,
+      null, castButtonSize,
+      this.props.config.moreDetailsScreen.color,
```
Comment (atopilskii) [inline sdk/react/panels/CastDevicesScreen.js]: and here

File: sdk/react/ooyalaSkinPanelRenderer.js (hunk 1 above)
Comment (atopilskii) [inline sdk/react/ooyalaSkinPanelRenderer.js]: Why do we use this "moreOptionsScreen" for cast button?

**Review by atopilskii — CHANGES_REQUESTED** [review]

**Review by isheftel-ooyala — COMMENTED** [review]
@atopilskii 
created separate cast options https://github.com/ooyala/android-sample-apps/pull/430

PR is updated, now we use colors from options in active and inactive states

**Review by atopilskii — APPROVED** [review]

**Pull request merged**